pynxtools/__init__.py

```diff
@@ -1,6 +1,5 @@
-"""init file
+"""init file"""
```

Comment:
```suggestion
"""
init file
"""
```


Do we want to have this docstring? I would suggest to remove it if it's just saying `init file`

Reply:
No, agree, apache default statement like for all nomad code files, will edit